service/src/test/java/uk/gov/hmcts/reform/fpl/service/DraftCMOServiceTest.java

```diff
@@ -83,7 +90,8 @@ void shouldReturnHearingDateDynamicListWhenCmoHasPreviousSelectedValue() {
     @Test
     void shouldReturnCaseManagementOrderWhenProvidedCaseDetails() {
         CaseDetails caseDetails = CaseDetails.builder()
-            .data(ImmutableMap.of("cmoHearingDateList", getDynamicList()))
+            .data(ImmutableMap.of("cmoHearingDateList", getDynamicList(), "allParties",
+                createDirection(null)))
```

Comment:
Couldn't we stick something in here? putting null in tests is always a bit confusing. Maybe we could extract the caseDetails creation if the null is in separate method it doesn't hurt that much.